### PR TITLE
Start the RCP for stage earlier

### PR DESCRIPTION
**Pull request by mokimo**
### Description
We want to keep `stage` clean in case of CSOs during RCPs to make it easier to contribute changes without pushing unwanted changes to production.

For this reason, we'd want to stop opening PRs to stage **before** the RCP so that nothing is open **during** the RCP.

**Test URLs:**
- Before: https://main--milo--mokimo.hlx.page/?martech=off
- After: https://keep-stage-clean--milo--mokimo.hlx.page/?martech=off

### CODE REVIEW

File: .github/workflows/merge-to-stage.js
Comment (overmyheadandbody) [inline .github/workflows/merge-to-stage.js:228]: As discussed, let's do 2, one to get SOT approvals, one to actually merge.

Also, I think most RCPs start on Sundays, even this approach might not be enough and we should instead look for the Wednesday before an RCP, so that we can eventually merge it on Thursday. But that may complicate things even further 🙂

Comment (mokimo) [reply .github/workflows/merge-to-stage.js]: The alternative is looking into the RCP dates and keeping a seperate list for stage and manually going through all the dates and editing them.

Contributions welcome 😁

**Review by overmyheadandbody — APPROVED** [review]
Agree, let's see how far this gets us and manually adjust dates if we run into issues.

**Review by robert-bogos — APPROVED** [review]
Looks good 👍

**Review by overmyheadandbody — APPROVED** [review]

**Pull request merged**
